change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -945,6 +1816,53 @@ PROCEDURE VALIDATECONFIG
    PARAMVALIDATIONERROR=TRUE
    PARAMVALIDATIONERRORCHR="PO Box Types Setup Incomplete"
   END
+[* If name matching name levels are set (not none) and no name types
+** are specified.
+*]
+ IF PARAMMATCHNAMELVL<>"NONE" AND
+    PARAMMATCHNMTYPES="NONE" THEN
+  DO
+   PARAMVALIDATIONERROR=TRUE
+   PARAMVALIDATIONERRORCHR="No name types set for name level matching"
+  END
+[* Check test accounts, if any, and validate each is 10 digits long
+*]
+ IF PARAMTESTACCTS<>"" THEN
+  DO
+   DATALINE=PARAMTESTACCTS
+   DELIMITER=","
+   CALL PARSEDATALINE
+
+   INVTESTACCTFOUND=FALSE
+   FOR TMPLOOP=1 TO DATAFIELDCOUNT
+    DO
+     IF LENGTH(DATALINEFIELD(TMPLOOP))<>10 THEN
+      INVTESTACCTFOUND=TRUE
+    END
+   IF INVTESTACCTFOUND=TRUE THEN
+    DO
+     PARAMVALIDATIONERROR=TRUE
+     PARAMVALIDATIONERRORCHR="Invalid test account found. Must be 10 digits long."
+    END
+  END
+[* If email is enabled, check for email detail lines and valid email address, if specified.
+*]
+ IF PARAMSENDEMAIL=TRUE THEN
+  DO
+   IF EMAILLINECOUNT=0 THEN
+    DO
+     PARAMVALIDATIONERROR=TRUE
+     PARAMVALIDATIONERRORCHR="No email detail lines specified for member email."
+    END
+   ELSE IF (PARAMEMAILFROM<>MEMBEREMAIL AND
+           (CHARACTERSEARCH(PARAMEMAILFROM,"@")=0 OR
+            CHARACTERSEARCH(PARAMEMAILFROM,".")=0)) THEN
```

Comment:
Should the check for a dot '.' also include the check that the dot is after the '@'? Otherwise wouldn't "tom.kainz@jackhenry" pass?

Reply:
Yes, I have updated the procedure.